### PR TITLE
fix: remove "Nameless" from  title in booking cancel page

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?
fixes the issue as mentioned in https://threads.com/34457815439?u=34440454662

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/pages/booking/[uid].tsx [hunk 1]
```diff
@@ -527,7 +527,7 @@ export default function Success(props: SuccessProps) {
                             </div>
                           )}
                           {bookingInfo?.attendees.map((attendee) => (
-                            <div key={attendee.name} className="mb-3 last:mb-0">
+                            <div key={attendee.name + attendee.email} className="mb-3 last:mb-0">
```
Comment (G3root) [inline apps/web/pages/booking/[uid].tsx:530]: fixes the same key warning when additional guests are present

File: apps/web/pages/booking/[uid].tsx [hunk 2]
```diff
@@ -485,7 +485,7 @@ export default function Success(props: SuccessProps) {
                       </>
                     )}
                     <div className="font-medium">{t("what")}</div>
-                    <div className="col-span-2 mb-6 last:mb-0">{eventName}</div>
+                    <div className="col-span-2 mb-6 last:mb-0">{props.bookingInfo.title}</div>
```
Comment (hariombalhara) [inline apps/web/pages/booking/[uid].tsx:488]: @G3root can you comment why nameless was coming and how switching from eventName to props.bookingInfo.title fixed it?

Also, is it something that recently regressed or like this always.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Need clarity on change

File: apps/web/pages/booking/[uid].tsx (hunk 2 above)
Comment (G3root) [reply apps/web/pages/booking/[uid].tsx:488]: previously event name has been derived from [here](https://github.com/calcom/cal.com/blob/1bf50e09cdeb4eb06683db080e9bd61abe991671/apps/web/pages/booking/%5Buid%5D.tsx#LL217-L220C20), in the event of cancelled booking , attendees are empty so it fallbacks to `Nameless`. using the title of the booking seems reliable here.

i think it is recently regressed.

**Review by CarinaWolli — APPROVED** [review]
Looks good to me 👍

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**